crates/uplc/src/interpreter.rs

```diff
@@ -0,0 +1,110 @@
+use crate::{
+    ast::{Constant, NamedDeBruijn, Term},
+    builtins::DefaultFunction,
+};
+
+enum StepKind {
+    BConst = 0,
```

Comment:
Do the `enum` variants need the discriminant (TIL the name of the number next to each)? I've never used them before in Rust.

Reply:
I'm not sure on that one. We might not even need this enum when all is said and done. I will find out as I get the final bits down.

Reply:
Cool cool. I realized after I commented that you're still WIP. I'll come back later :)